src/components/widgets/PAutoComplete.js

```diff
@@ -1,26 +1,39 @@
 import { PInput, PListItem, PList } from '../elements'
 import React from 'react'
+
+const capitalizeFirstLetter = string => string.charAt(0).toUpperCase() + string.slice(1)
+
 export default (props) => {
   return (
-    <div>
+    <div
+      className="auto-complete"
+    >
       <PInput
+        disabled={props.loading}
         value={props.value}
         onChange={(val) => props.onChange && props.onChange(val)}
       ></PInput>
       {props.suggestions && props.suggestions.length > 0 ? (
         <PList>
-          {props.suggestions.map((item) => (
+          {
+            props.loading && <p>Loading...</p>
+          }
+          {!props.loading && props.suggestions.map((item) => (
             <PListItem
               key={item}
               onPress={() => {
                 props.onSelect && props.onSelect(item)
               }}
-            >
-              {item}
-            </PListItem>
+              text={{
+                highlighted: props.value,
+                normal: item.replace(capitalizeFirstLetter(props.value), ''),
```

Comment:
Change inside text from ChildNode to Attribute would not be a good approach to highlight the text.
Use ChildNode can open a gate to use any thing as an inner text such as a button, label or a complex elements.

Reply:
I can't agree more!